Source/HTTP/Curl/CurlEasyRequest.cpp

```diff
@@ -40,12 +40,30 @@ Result<HC_UNIQUE_PTR<CurlEasyRequest>> CurlEasyRequest::Initialize(HCCallHandle
     void* clientRequestBodyReadCallbackContext{};
     RETURN_IF_FAILED(HCHttpCallRequestGetRequestBodyReadFunction(hcCall, &clientRequestBodyReadCallback, &bodySize, &clientRequestBodyReadCallbackContext));
 
+// Specify libcurl progress callback and create libcurl progress callback for non-GDK platforms since XCurl doesn't support libcurl progress callback
+#if HC_PLATFORM != HC_PLATFORM_GDK
```

Comment:
is the alternative implementation for XCurl already done somewhere that I'm missing or is that still pending?

Reply:
It's already implemented in lines 259 and 397, instead of using the built-in progress callback from XCurl, we're reporting progress from Write and Read callbacks from WinHttp.